core/src/main/java/com/google/instrumentation/stats/RpcConstants.java

```diff
@@ -40,86 +40,117 @@
   // RPC client {@link MeasurementDescriptor}s.
   public static final MeasurementDescriptor RPC_CLIENT_ERROR_COUNT =
       MeasurementDescriptor.create(
-          "/rpc/client/error_count",
+          "/grpc.io/client/error_count",
```

Comment:
Should these names have leading slashes?  I noticed that the names in https://github.com/google/instrumentation-proto/pull/24 just start with "grpc.io".